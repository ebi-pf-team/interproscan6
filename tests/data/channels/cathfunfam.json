{"929DAA6772F74C7C5B5FB5B454E3D463":{"2.170.130.10-FF-000010":{"modelAccession":"2.170.130.10-FF-000010","evalue":6.6E-37,"score":128.2,"bias":0.3,"signature":{"accession":"G3DSA:2.170.130.10:FF:000010","name":null,"description":null,"type":null,"signatureLibraryRelease":{"library":"CATH-FunFam","version":null},"entry":null},"locations":[{"start":51,"end":177,"hmmStart":1,"hmmEnd":131,"hmmLength":132,"hmmBounds":"[.","envelopeStart":51,"envelopeEnd":177,"evalue":1.7E-36,"score":126.9,"bias":null,"queryAlignment":null,"targetAlignment":null,"sequenceFeature":null,"pvalue":null,"motifNumber":null,"level":null,"cigarAlignment":null,"fragments":[{"start":51,"end":177,"dcStatus":"CONTINUOUS"}],"sites":[],"representative":false,"included":true}],"included":true,"representativeInfo":null,"treegrafter":null,"graphscan":null}},"C84B0963DF32B12896E7154FF7B073AD":{"3.40.1170.10-FF-000001":{"modelAccession":"3.40.1170.10-FF-000001","evalue":7.9E-52,"score":176.2,"bias":0.0,"signature":{"accession":"G3DSA:3.40.1170.10:FF:000001","name":null,"description":null,"type":null,"signatureLibraryRelease":{"library":"CATH-FunFam","version":null},"entry":null},"locations":[{"start":7,"end":126,"hmmStart":4,"hmmEnd":118,"hmmLength":119,"hmmBounds":"..","envelopeStart":7,"envelopeEnd":126,"evalue":2.0E-51,"score":175.0,"bias":null,"queryAlignment":null,"targetAlignment":null,"sequenceFeature":null,"pvalue":null,"motifNumber":null,"level":null,"cigarAlignment":null,"fragments":[{"start":7,"end":126,"dcStatus":"CONTINUOUS"}],"sites":[],"representative":false,"included":true}],"included":true,"representativeInfo":null,"treegrafter":null,"graphscan":null},"1.10.1420.10-FF-000002":{"modelAccession":"1.10.1420.10-FF-000002","evalue":2.5E-47,"score":161.8,"bias":0.4,"signature":{"accession":"G3DSA:1.10.1420.10:FF:000002","name":null,"description":null,"type":null,"signatureLibraryRelease":{"library":"CATH-FunFam","version":null},"entry":null},"locations":[{"start":400,"end":516,"hmmStart":3,"hmmEnd":117,"hmmLength":117,"hmmBounds":".]","envelopeStart":400,"envelopeEnd":516,"evalue":5.9E-47,"score":160.6,"bias":null,"queryAlignment":null,"targetAlignment":null,"sequenceFeature":null,"pvalue":null,"motifNumber":null,"level":null,"cigarAlignment":null,"fragments":[{"start":400,"end":516,"dcStatus":"CONTINUOUS"}],"sites":[],"representative":false,"included":true}],"included":true,"representativeInfo":null,"treegrafter":null,"graphscan":null}}}