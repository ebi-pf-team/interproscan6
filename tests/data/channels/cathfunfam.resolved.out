# Generated by cath-resolve-hits v0.16.10-0-g99edb28, one of the cath-tools (https://github.com/UCLOrengoGroup/cath-tools)
#FIELDS query-id match-id score boundaries resolved aligned-regions cond-evalue indp-evalue
929DAA6772F74C7C5B5FB5B454E3D463 2.170.130.10-FF-000010 126.9 51-177 51-177 1-62,51-112;64-93,113-142;98-131,143-176 2.6e-41 1.7e-36
C84B0963DF32B12896E7154FF7B073AD 3.40.1170.10-FF-000001 175 7-126 7-126 4-94,12-102;96-118,103-125 3e-56 2e-51
C84B0963DF32B12896E7154FF7B073AD 1.10.1420.10-FF-000002 160.6 400-516 400-516 3-117,402-516 9e-52 5.9e-47
